open_bus_stride_api/DESCRIPTION.md

```diff
@@ -0,0 +1,16 @@
+### Follow planned and real-time bus statistics 🚌
+
+Looking for a specific use-case?
+* Take a look at the **user cases** APIs below.
+* Try one of our existing notebooks-
+  * [Load route rides to dataframe](https://mybinder.org/v2/gh/hasadna/open-bus-stride-client/HEAD?labpath=notebooks%2FLoad%20route%20rides%20to%20dataframe.ipynb)
+  * [getting all arrivals to all stops of a given line on a given day](https://mybinder.org/v2/gh/hasadna/open-bus-stride-client/HEAD?labpath=notebooks%2Fgetting%20all%20arrivals%20to%20all%20stops%20of%20a%20given%20line%20in%20a%20given%20day.ipynb)
+  * [load siri vehicle locations to pandas dataframe](https://mybinder.org/v2/gh/hasadna/open-bus-stride-client/main?labpath=notebooks%2Fload%20siri%20vehicle%20locations%20to%20pandas%20dataframe.ipynb)
```

Comment:
the list of notebooks may change over time and then we may get broken links here. we might add / remove or rename notebooks, so it's best not to list them here, instead you should link to the list in the python client - https://github.com/hasadna/open-bus-stride-client/blob/main/README.md#using-the-interactive-jupyter-notebooks

Reply:
Agree, the initial thought was that maybe people will be too lazy to search in the repo 😅